mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -2097,6 +2085,43 @@ void ModuleImport::convertParameterAttributes(llvm::Function *func,
       builder.getArrayAttr(convertParameterAttribute(llvmResAttr, builder)));
 }
 
+namespace impl {
+template <typename Op>
+LogicalResult convertCallBaseAttributes(llvm::CallBase *inst, Op op) {
+  op.setCConv(convertCConvFromLLVM(inst->getCallingConv()));
+  return success();
+}
+} // namespace impl
```

Comment:
```suggestion
template <typename Op>
static LogicalResult convertCallBaseAttributes(llvm::CallBase *inst, Op op) {
  op.setCConv(convertCConvFromLLVM(inst->getCallingConv()));
  return success();
}
```
nit: AFAIK the preferred way for such helper templates in cpps is to use static template functions. Or did this not work for some reason?

Reply:
fixed in 1c441a8a68d66e4bf64a91fe98160417d8132607